dynalab/tasks/annotation_mock_data.py

```diff
@@ -40,50 +40,49 @@ def generate_context_string_selection_mock_data(annotation, name_to_annotation_d
     return [source_str[0:10] for source_str in source_data]
 
 
-def generate_conf_mock_data(annotation=None, name_to_annotation_dict=None):
-    return [random.random() for _ in range(3)]
+def generate_prob_mock_data(annotation, name_to_annotation_dict):
 
+    if annotation.get("single_prob", False):
+        return [random.random() for _ in range(3)]
+    else:
+        mock_data = []
+        source_reference_name = annotation["reference_name"]
+        source_annotation = name_to_annotation_dict[source_reference_name]
+        labels = source_annotation["labels"]
 
-def generate_multiclass_probs_mock_data(annotation, name_to_annotation_dict):
-    mock_data = []
-    source_reference_name = annotation["constructor_args"]["reference_name"]
-    source_annotation = name_to_annotation_dict[source_reference_name]
-    labels = source_annotation["constructor_args"]["labels"]
-
-    for _ in range(3):
-        probs_dict = {}
-        probs_sum = 0
-        for label in labels:
-            probs_dict[label] = random.random()
-            probs_sum += probs_dict[label]
+        for _ in range(3):
+            probs_dict = {}
+            probs_sum = 0
+            for label in labels:
+                probs_dict[label] = random.random()
+                probs_sum += probs_dict[label]
 
-        # normalize
-        for label in labels:
-            probs_dict[label] /= probs_sum
+            # normalize
+            for label in labels:
+                probs_dict[label] /= probs_sum
```

Comment:
probably not a problem but is it possible that the values in `probs_dict` are `int`'s because I think `probs_sum` here is an int so then this might result in integer division instead of float division

Reply:
Doesn't python3 not have an issue with int division, though?